src/software/ai/navigator/path_planner/hrvo/hrvo_simulator.cpp

```diff
@@ -152,10 +152,15 @@ void HRVOSimulator::doStep(Duration time_step)
 
 void HRVOSimulator::visualize(unsigned int robot_id, TeamColour friendly_team_colour)
 {
+    if (robots.empty())
+    {
+        return;
+    }
```

Comment:
Curious why this function would be called if there were no robots in the World? Since we didn't have this problem before. Does the log get spammed throughout the test running or just at the start of it?